src/routes/profile/index.svelte

```diff
@@ -3,7 +3,12 @@
 
 	import { onMount } from 'svelte';
 
-	import { discord, getDiscordProfilePicture, getFormattedDate } from '../../stores/discord';
+	import {
```

Comment:
lint prob wont like this

Reply:
Ran format and lint and everything does the work now locally